addresspoints/src/main/scala/grasshopper/addresspoints/model/Status.scala

```diff
@@ -0,0 +1,3 @@
+package grasshopper.addresspoints.model
+
+case class Status(status: String, service: String, time: String, host: String)
```

Comment:
Regarding upSince, yeah, I think it's a good addition. With respect to creating a module that all projects depend on (for classes like Status), I think it produces tightly coupled code between microservices. Yes, there is some duplication of code, but more flexibility